docs/data/sdks/java/index.md

```diff
@@ -46,9 +46,33 @@ Amplitude client = Amplitude.getInstance("YOUR_INSTANCE_NAME");
 client.init("YOUR_API_KEY");
 ```
 
-### Configure batching behavior
-
-To support high performance environments, the SDK sends events in batches. Every event logged by `logEvent` method is queued in memory. Events are flushed in batch in background. You can customize batch behavior with `setEventUpdloadThreshfold` and `setEventUploadPeriodMillis`.
+### Configuration
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `setServerUrl` | The server url events are uploaded to. | `https://api2.amplitude.com/2/httpapi` |
+    | `useBatchMode` |  Whether to use [batch api](../../../analytics/apis/batch-event-upload-api/#batch-event-upload). By default, the SDK will use the default `serverUrl` | `false` |
+    | `setLogMode` |  The level at which to filter out debug messages. | `LogMode.ERROR` |
+    | `setEventUploadThreshold` |  SDK will attempt to upload once unsent event count exceeds the event upload threshold or reach eventUploadPeriodSeconds interval. | `10` |
+    | `setEventUploadPeriodMillis` | The amount of time SDK will attempt to upload the unsent events to the server or reach eventUploadThreshold threshold. | `10 seconds` |
+    | `setCallbacks` | Event callback which are triggered after event sent. | `null`|
+    | `setProxy` | Custom proxy for https requests. | `Proxy.NO_PROXY`|
+    | `setFlushTimeout` | Events flushing thread timeout in milliseconds. | `0` |
+    | `setOptions` | A dictionary of key-value pairs that represent additional instructions for server save operation.| Please check [available options](/#options). |
+
+### Options
+
+???config "Availabe Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `Options.setMinIdLength` | Set the minimum length for user id or device id. | 5 |
+    | `Options.setHeaders` | Set the custom headers. | `{"Content-Type", "application/json","Accept", "application/json"}` |
```

Comment:
Minor, missing space `json","Accept`, also to confirm, should this be Map or JSON? (https://github.com/amplitude/Amplitude-Java/blob/0219df6aa61e1414326fe759ab1dbd569daa83c7/src/main/java/com/amplitude/Options.java#L23)